yt/visualization/plot_window.py

```diff
@@ -1119,6 +1119,15 @@ def annotate_clear(self, index=None):
         self.setup_callbacks()
         return self
 
+    def annotate_list(self):
```

Comment:
Ok, I see that you're matching the `annotate_clear` method in the way that this operates on all annotations. That said, I'm wondering if it's maybe clearer to name it something like `list_annotations`? I feel like that might be more intuitive when looking through different operations one could do on a plot. If I were casually using yt, I'd think that `annotate_list` would be a specific annotation of a list object, just like `annotate_scale` annotates a scale.

Reply:
I agree, it would be clearer, and I'm happy to change the name, but I was trying to match the convention already in place.  E.g., `clear_annotations()` would also be more clear than `annotations_clear()`